back/node_watcher/src/prismaWriter.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018-2019 @paritytech/nomidot authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { prisma } from '../generated/prisma-client';
+import { BlockInfoFromNode } from './types';
+
+/*
+* Handles async/await writing to PSQL with Prisma GraphQL client
+* Basically subscriptions to node could move on before async write resolves
+* so we put them in a queue here.
```

Comment:
I would be way more easier to remove the queue, never subscribe to tip, and use `.at` everywhere

Reply:
ah, forgot about `.at`.